google-cloud-spanner/src/main/java/com/google/cloud/spanner/AbstractStructReader.java

```diff
@@ -19,9 +19,9 @@
 import static com.google.common.base.Preconditions.checkState;
 
 import com.google.cloud.ByteArray;
-import com.google.cloud.Date;
 import com.google.cloud.Timestamp;
 import java.util.List;
+import org.joda.time.LocalDate;
```

Comment:
We should be using date types from threetenbp, not joda.

Reply:
Oh! I saw we're using Joda in a few places, but those are internal. Will fix this.